documentation/configuration.md

```diff
@@ -21,5 +21,19 @@ metrics: {
             enabled: true
         }
     }
+
+    jolokia: {
```

Comment:
I thought about that a bit as well, in the end `reporters` didn't seem like the right section since this  is an API for incoming requests and not something the reports out metrics on a given schedule.